tests/js-modules/modules.py

```diff
@@ -425,6 +425,7 @@ def test_dynamic_endpoints(network, args):
 
 
 @reqs.description("Test basic Node.js/npm app")
+@reqs.not_snp()
```

Comment:
Please add reason why so that we can keep track of it.

Reply:
Will do, I'll wait for the current CI run to complete before pushing since it's midway through